packages/lit-dev-content/site/docs/v1/lit-html/release-notes.md

```diff
@@ -4,6 +4,8 @@ eleventyNavigation:
   key: Release notes
   parent: lit-html
   order: 10
+versionLinks:
+  v2: releases/upgrade/
```

Comment:
ok for now, but we need to finish out the release notes section on the new docs

Reply:
+1